go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -821,3 +818,16 @@ func extractNamesFromTablesList(tables []*Table) []string {
 	}
 	return tableNames
 }
+
+func (se *Engine) ResetSequences(tables []string) {
+	se.mu.Lock()
+	defer se.mu.Unlock()
+	for _, tableName := range tables {
+		if table, ok := se.tables[tableName]; ok {
+			if table.SequenceInfo != nil {
+				log.Infof("Resetting sequence info for table %v: %s", tableName, table.SequenceInfo)
+				table.SequenceInfo.Reset()
+			}
+		}
+	}
```

Comment:
If table is not present in here, should we return an error?

Reply:
Done